functions/mqhandlers.go

```diff
@@ -209,12 +208,11 @@ func HostUpdate(client mqtt.Client, msg mqtt.Message) {
 		config.UpdateServer(serverName, *server)
 		config.WriteNodeConfig()
 		config.WriteServerConfig()
-		logger.Log(1, "added node for network", hostUpdate.Node.Network, "on server", serverName)
+		slog.Info("added node to network", "network", hostUpdate.Node.Network, "server", serverName)
 		clearRetainedMsg(client, msg.Topic()) // clear message before ACK
 		if err = PublishHostUpdate(serverName, models.Acknowledgement); err != nil {
-			logger.Log(0, "failed to response with ACK to server", serverName)
+			slog.Error("failed to response with ACK to server", "server", serverName, "error", err)
 		}
-		restartDaemon = true
```

Comment:
set `resetInterface` to true here otherwise,  interface won't be configured for the new network